src/synced_commands.cpp

```diff
@@ -478,7 +478,14 @@ SYNCED_COMMAND_HANDLER_FUNCTION(debug_unit, child,  use_undo, /*show*/, /*error_
 		return false;
 	}
 	if (name == "advances" ) {
-		int int_value = std::stoi(value);
+		int int_value = 0;
+		try {
+			int_value = std::stoi(value);
+		}
+		catch (const std::invalid_argument&) {
```

Comment:
I think usually catch is put on the same line as the closing brace.

Reply:
I was thinking that too, actually.

Reply:
MSVC did the editing for me, claiming that it found a 'clang formatting file' or some such.

Reply:
Perhaps that clang format file should be fixed then since a quick grep shows the majority of lines not using that formatting.

Reply:
Historically I notice people use whatever formatting - it's just that recent versions of MSVC seems to detect and use this. I generally try to stick with whatever is already present - certainly if I had the choice I would not use this coding style.

Reply:
Well, it looks like `src/.clang-format` has `BraceWrapping: BeforeCatch: false` which should mean it doesn't wrap for catch and so puts it on the same line as the closing brace. Do you know what 'clang formatting file' was used?

Reply:
No, but MSVC does a lot of auto-formatting stuff in the background that I don't really pay attention to it any more (I find it rather annoying sometimes). As I said, I just try to stick to what's already in place. I don't think it's worth the effort to investigate this further.